src/main/java/slimeknights/tconstruct/tables/block/entity/table/TinkerStationBlockEntity.java

```diff
@@ -132,9 +133,9 @@ public ItemStack calcResult(@Nullable Player player) {
         }
 
         // try for UI errors
-        RecipeResult<ItemStack> validatedResult = recipe.getValidatedResult(this.inventoryWrapper);
+        RecipeResult<LazyToolStack> validatedResult = recipe.getValidatedResult(this.inventoryWrapper);
         if (validatedResult.isSuccess()) {
-          result = validatedResult.getResult();
+          result = validatedResult.getResult().getStack();
```

Comment:
Resolving the lazy tool stack immediately defeats the purpose of using it in the first place.

Take a look at the modifier worktable block entity, that shows how to properly use this utility.

Reply:
On account of the call later on to TooltipUtil.setDisplayName(ItemStack tool, String name), won't I have to resolve it? Otherwise, I can make a new method for LazyToolStacks instead to do the same thing to the nbt

Reply:
Let me know your thoughts but the [commit](https://github.com/SlimeKnights/TinkersConstruct/pull/5282/commits/651ef6b42ce0687ef40979769fc0bec84ac7707b) I just pushed has that kind of idea. I can revert later if needed